locale/fr/docs/index.md

```diff
@@ -5,23 +5,23 @@ labels:
   lts: LTS
 ---
 
-# About Docs
+# À propos de Docs
 
-There are several types of documentation available on this website:
+Il existe plusieurs types de documentation disponibles sur ce site web:
```

Comment:
```suggestion
Il existe plusieurs types de documentations disponibles sur ce site web:
```
I am not 100% sure on this one

Reply:
Would you rather say `plusieurs types de cheval` or `plusieurs types de chevaux`? I'd say both are acceptable, the plural form is maybe more prevalent though.

Reply:
I used http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?id=1596#:~:text=On%20peut%20n%C3%A9anmoins%20retenir%20quelques,une%20valeur%20abstraite%20non%20d%C3%A9nombrable. ```On peut néanmoins retenir quelques principes. Après les mots genre, espèce, variété, classe, forme, sorte, catégorie, type, etc., au pluriel, le complément est généralement au pluriel s’il se réfère à une réalité concrète ou que l’on peut compter, et au singulier s’il désigne une valeur abstraite non dénombrable.``` But I am not fully certain either ^^